eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Newpp.hs

```diff
@@ -106,29 +99,21 @@ newPpTransition ::
   TransitionRule (ShelleyNEWPP era)
 newPpTransition = do
   TRC
-    ( NewppEnv dstate pstate utxoSt acnt,
+    ( NewppEnv dstate pstate utxoSt _,
       NewppState pp ppupSt,
       ppNew
       ) <-
     judgmentContext
 
   case ppNew of
     Just ppNew' -> do
-      let Coin oblgCurr = obligation pp (rewView (dsUnified dstate)) (psStakePoolParams pstate)
-          Coin oblgNew = obligation ppNew' (rewView (dsUnified dstate)) (psStakePoolParams pstate)
-          diff = oblgCurr - oblgNew
-          Coin availableReserves = availableAfterMIR ReservesMIR acnt (dsIRewards dstate)
```

Comment:
How come we don't have to care about  MIR  here? Does this check follow as a consequence of the new implementation?